### PR TITLE
Shapes layer internal renaming

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR just does some internal renaming to make the Shapes layer consistent with the naming conventions adopted in #1010, #1033. It might cause some conflicts with #898 but they should be pretty easy to resolve. After this PR merges I'll work on actually doing the mouse bindings refactor, but that will be easier to review with all this renaming already done. Note this breaks nothing with our public API. I will make a new PR in parallel to propagate the renaming changes to the rest of the repo.

### CODE REVIEW

File: napari/layers/shapes/_shapes_list.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-from .mesh import Mesh
-from ._constants import shape_classes, ShapeType
+from ._mesh import Mesh
+from ._shapes_models import Shape, Line, Path
+from ._shapes_utils import inside_triangles, triangles_intersect_box
+from ._shapes_constants import shape_classes, ShapeType
 
 
-class ShapeList:
+class ShapesList:
```
Comment (jni) [inline napari/layers/shapes/_shapes_list.py]: Not in love with this change...? ShapeList matches LayerList which matches "car collection" which matches "card deck"...

**Review by jni — CHANGES_REQUESTED** [review]
@sofroniewn honestly I generally prefer the the singular name, but I specifically prefer it in the NounList form — which matches English usage in general, "Item Collection" rather than "Items Collection". For the other stuff, it's internal so I don't mind so much.

File: napari/layers/shapes/_shapes_list.py (hunk 1 above)
Comment (sofroniewn) [reply napari/layers/shapes/_shapes_list.py]: I will revert

**Comment by sofroniewn** [comment]
This has now been updated. `ShapeList` is singular, but everything else is plural, `add_shapes`, `Shapes` etc. This is keeping with the current API. Changing it would be a breaking change. Is this now ok @jni?

**Review by jni — APPROVED** [review]
Yes yes, perfect, a "ShapeList" contains "Shapes", this is totally fine and sensical and non-breaking changes are good. =)

**Pull request merged**